Documentation/contributing/testing/e2e.rst

```diff
@@ -637,6 +637,8 @@ configuration options that can be passed as environment variables:
 +======================+===================+==============+==================================================================+
 | K8S\_NODES           | 2                 | 0..100       | Number of Kubernetes nodes in the cluster                        |
 +----------------------+-------------------+--------------+------------------------------------------------------------------+
+| NO_CILIUM_ON_NODE[S] | none              | *            | Comma-separated list of K8s nodes that should not run Cilium     |
```

Comment:
Nit: the star is rendered as a list item bullet, you might want to escape it with a backslash.
![image](https://user-images.githubusercontent.com/17001771/166491845-78542de0-182a-419c-bc66-43b4efb0c9e4.png)

Reply:
Ah indeed - thanks, fixed. Also fixed up the same issue for `SERVER_BOX`.